AdobeIms/Model/GetToken.php

```diff
@@ -87,7 +87,7 @@ public function execute(string $code): TokenResponseInterface
 
         if (empty($tokenResponse->getAccessToken()) || empty($tokenResponse->getRefreshToken())) {
             throw new AuthorizationException(
-                __('Authentication is failing. Error code: %error', ['error' => $tokenResponse->getError()])
+                __('Login failed. Check if the Secret Key entered correctly and try again.')
```

Comment:
This specific exception message is only for `$tokenResponse->getError() === 'invalid_client'`